projects/storefrontapp-e2e-cypress/cypress/integration/vendor/cdc/b2b/b2b-manage-users-e2e-spec.ts

```diff
@@ -0,0 +1,66 @@
+import { loginUser } from '../../../../helpers/checkout-flow';
```

Comment:
@anjana-bl as discussed, please rename this file as `projects/storefrontapp-e2e-cypress/cypress/integration/vendor/cdc/b2b/b2c-scenarios.core-e2e-spec.ts`

Reply:
done